paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -139,14 +139,16 @@ internal abstract class BaseSheetViewModel(
     private val savedSelection: StateFlow<SavedSelection?> = savedStateHandle
         .getStateFlow<SavedSelection?>(SAVE_SAVED_SELECTION, null)
 
-    protected val backStack = MutableStateFlow<List<PaymentSheetScreen>>(emptyList())
+    protected val backStack = MutableStateFlow<List<PaymentSheetScreen>>(
+        value = listOf(PaymentSheetScreen.Loading),
+    )
 
-    val currentScreen: StateFlow<PaymentSheetScreen?> = backStack
-        .map { it.lastOrNull() }
+    val currentScreen: StateFlow<PaymentSheetScreen> = backStack
+        .map { it.last() }
```

Comment:
Do we have tests to prevent us from making the backstack empty?

Reply:
Amended an existing test to keep an eye on this behavior.